XenAdmin/Dialogs/CallHome/CallHomeOverviewDialog.cs

```diff
@@ -191,6 +191,7 @@ private void RefreshDetailsPanel()
 
             healthCheckStatusPanel.Visible = poolRow.Pool.CallHomeSettings.Status == CallHomeStatus.Enabled;
             notEnrolledPanel.Visible = poolRow.Pool.CallHomeSettings.Status != CallHomeStatus.Enabled;
+            uploadRequestLinkLabel.Enabled = poolRow.Pool.CallHomeSettings.CanRequestNewUpload;
```

Comment:
When clicked, instead of disabling it, it would be nice to change the link to an unclickable text saying "Requested an upload at....".